packages/core/lib/modules/wallet.js

```diff
@@ -174,6 +201,18 @@ export class Wallet {
     return result[0];
   }
 
+  static async create({walletId, json, password} = {}): Wallet {
+    const wallet = new Wallet({walletId});
+
+    await wallet.load();
+
+    if (json) {
+      await WalletRpc.importWallet(json, password);
```

Comment:
is there some magic happening here? or rather, how does `WalletRpc.importWallet` affect `wallet`? Or does it? I guess I"m a bit confused as to what the expected effect of this call is

Reply:
This is handled by the universal wallet sdk, it will decrypt the json and push the documents to the wallet storage.

Reply:
so why do we need the `wallet` object? It's not obviously involved in that. i.e. it's not passed to the WalletRpc call, so how is that connected?

Reply:
Okay, got it now.